app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z0-9\-.]+\.[a-z]+\z/i.freeze
```

Comment:
Sao phải thêm `.freezee` thế

Reply:
Dạ chị, tại vì khi em chạy lệnh rubocop thì nó báo " Freeze mutable objects assigned to constants". Vì vậy em phải .freeze giá trị của hằng số này ạ!

Reply:
Chị cho em trả lời lại :v. Trong Ruby, vì các hằng số có thể thay đổi, nên ta cần dùng .freeze để giá trị này không bị thay đổi, nếu như cố tình thay đổi thì hệ thống sẽ báo lỗi

Reply:
trả lời thế này sao còn xóa code `.freeze` vậy